lib/lumberjack/client.rb

```diff
@@ -4,6 +4,7 @@
 require "thread"
 require "openssl"
 require "zlib"
+require "json"
```

Comment:
I recommend we can override it, in our case jrjackson is more performant so we should be using it in the server.